build/template-build-and-prep-automation.yaml

```diff
@@ -96,8 +96,73 @@ steps:
     Contents: '**/bin/**/*'
     TargetFolder: '$(build.artifactstagingdirectory)/DeviceAuth'
 
+- task: CopyFiles@2
+  displayName: 'Stage WAM DevApp Dlls'
+  inputs:
+    SourceFolder: tests/devapps/wam
+    Contents: '**/bin/**/**'
+    TargetFolder: '$(build.artifactstagingdirectory)/WAM'
```

Comment:
Not sure you need to stage these Dlls here. This directory is used for uploading files for other build phases so they can be used for tests/signing. We try not to add too many files here because it can slow down the build quite a bit since these files need to be uploaded and downloaded on other agents. Do these Dlls need other be used by other phases for anything? If not, you can move them to a different local directory. Something like `Build.SourcesDirectory/Dlls`. Seems like you are performing the validation on the same agent anyway.

Reply:
fixed it